Experimental/DistributedAuthoritySample/Assets/Scripts/Player/AvatarTransform.cs

```diff
@@ -0,0 +1,272 @@
+        SerializedProperty m_AirControlFactor;
+        SerializedProperty m_JumpImpulse;
+        SerializedProperty m_CustomGravityMultiplier;
+        SerializedProperty m_RotationSpeed;
+        SerializedProperty m_GroundCheckDistance;
+
+        public override void OnEnable()
+        {
+            m_Rigidbody = serializedObject.FindProperty(nameof(AvatarTransform.m_Rigidbody));
+            m_PlayerInput = serializedObject.FindProperty(nameof(AvatarTransform.m_PlayerInput));
+            m_AvatarInputs = serializedObject.FindProperty(nameof(AvatarTransform.m_AvatarInputs));
+            m_WalkSpeed = serializedObject.FindProperty(nameof(AvatarTransform.m_WalkSpeed));
+            m_SprintSpeed = serializedObject.FindProperty(nameof(AvatarTransform.m_SprintSpeed));
+            m_Acceleration = serializedObject.FindProperty(nameof(AvatarTransform.m_Acceleration));
+            m_DragCoefficient = serializedObject.FindProperty(nameof(AvatarTransform.m_DragCoefficient));
+            m_AirControlFactor = serializedObject.FindProperty(nameof(AvatarTransform.m_AirControlFactor));
+            m_JumpImpulse = serializedObject.FindProperty(nameof(AvatarTransform.m_JumpImpusle));
+            m_CustomGravityMultiplier = serializedObject.FindProperty(nameof(AvatarTransform.m_CustomGravityMultiplier));
+            m_RotationSpeed = serializedObject.FindProperty(nameof(AvatarTransform.m_RotationSpeed));
+            m_GroundCheckDistance = serializedObject.FindProperty(nameof(AvatarTransform.m_GroundCheckDistance));
+
+            base.OnEnable();
+        }
+
+        public override void OnInspectorGUI()
+        {
+            var avatarTransform = target as AvatarTransform;
+            avatarTransform.PropertiesVisible = EditorGUILayout.BeginFoldoutHeaderGroup(avatarTransform.PropertiesVisible, $"{nameof(AvatarTransform)} Properties");
+            if (avatarTransform.PropertiesVisible)
+            {
+                EditorGUILayout.EndFoldoutHeaderGroup();
+                EditorGUILayout.PropertyField(m_Rigidbody);
+                EditorGUILayout.PropertyField(m_PlayerInput);
+                EditorGUILayout.PropertyField(m_AvatarInputs);
+                EditorGUILayout.PropertyField(m_WalkSpeed);
+                EditorGUILayout.PropertyField(m_SprintSpeed);
+                EditorGUILayout.PropertyField(m_Acceleration);
+                EditorGUILayout.PropertyField(m_DragCoefficient);
+                EditorGUILayout.PropertyField(m_AirControlFactor);
+                EditorGUILayout.PropertyField(m_JumpImpulse);
+                EditorGUILayout.PropertyField(m_CustomGravityMultiplier);
+                EditorGUILayout.PropertyField(m_RotationSpeed);
+                EditorGUILayout.PropertyField(m_GroundCheckDistance);
+            }
+            else
+            {
+                EditorGUILayout.EndFoldoutHeaderGroup();
+            }
+
+            EditorGUILayout.Space();
+            base.OnInspectorGUI();
+        }
+    }
```

Comment:
this is using IMGUI, which should not be used anymore for new code (be it runtime or editor), see [this thread ](https://unity.slack.com/archives/C06TQ9LFP/p1719948221825629?thread_ts=1719947570.042579&cid=C06TQ9LFP) for more infromation (can't post more details here as this repo is public)

Reply:
Done for now, but will include namespaces in a subsequent PR.